### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ilynivin**
### Checklist
  - [X] I Created A Toc In readme
  - [X] I  Created A Contributors Image and Hooked it on Readme.md

### Description
Updated Readme.md and Added Contribution Heading on the readme

### CODE REVIEW

**Review by RyanLua — CHANGES_REQUESTED** [review]
I've reviewed your changes and spotted some minor mistakes and suggestions.

- Lines 24 and 26 have spaces after the word "Usage".
- Line 31 is an extra line and should be deleted.
- Line 29 and 197 shouldn't have emojis to be consistent with the rest of the Readme.

These are just my suggestions; you don't have to follow all of them through.

File: README.md [hunk 1]
```diff
@@ -15,6 +15,20 @@ Over 2400 Free SVG icons for popular brands. See them all on one page at <a href
 <a href="https://opencollective.com/simple-icons"><img src="https://img.shields.io/opencollective/all/simple-icons?logo=opencollective" alt="Backers and sponsors on Open Collective"/></a>
 </p>
 
+## Table of Contents 
+- [Table of Contents](#table-of-contents)
+- [Usage](#usage)
+  - [General Usage](#general-usage)
+  - [CDN Usage](#cdn-usage)
+    - [CDN with colors](#cdn-with-colors)
+  - [Node Usage ](#node-usage-)
```
Comment (RyanLua) [inline README.md:24]: Remove extra space.
```suggestion
  - [Node Usage](#node-usage-)
```

File: README.md [hunk 2]
```diff
@@ -15,6 +15,20 @@ Over 2400 Free SVG icons for popular brands. See them all on one page at <a href
...
+    - [TypeScript Usage ](#typescript-usage-)
+  - [PHP Usage ](#php-usage-)
```
Comment (RyanLua) [inline README.md:26]: Remove extra space.
```suggestion
  - [PHP Usage](#php-usage-)
```

File: README.md [hunk 3]
```diff
@@ -15,6 +15,20 @@ Over 2400 Free SVG icons for popular brands. See them all on one page at <a href
...
+- [Third-Party Extensions](#third-party-extensions)
+- [Contribute](#contribute)
+- [Contributors 💕](#contributors-)
```
Comment (RyanLua) [inline README.md:29]: Remove emoji for consistency.
```suggestion
- [Contributors](#contributors-)
```

File: README.md [hunk 4]
```diff
@@ -15,6 +15,20 @@ Over 2400 Free SVG icons for popular brands. See them all on one page at <a href
...
+
+
```
Comment (RyanLua) [inline README.md:31]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -178,3 +192,14 @@ echo file_get_contents('path/to/package/icons/simpleicons.svg');
 Information describing how to contribute can be found in the file [CONTRIBUTING.md](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md)
 
 [slug]: https://github.com/simple-icons/simple-icons/blob/master/slugs.md
+
+
+## Contributors 💕
```
Comment (RyanLua) [inline README.md:197]: Remove emoji for consistency.
```suggestion
## Contributors
```

**Review by RyanLua — CHANGES_REQUESTED** [review]

**Comment by LitoMore** [comment]
Sorry, we don't want this. GitHub already has a ToC feature. See https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/.

On the other hand, those anchors you are trying to add won't work correctly on the description of https://npmjs.com/simple-icons.

**Comment by ilynivin** [comment]
Ok . So I am closing this pull request

**Pull request closed without merging**